ecosystem/sep-0006.md

```diff
@@ -67,6 +67,50 @@ Valid CORS headers are necessary to allow web clients from other sites to use th
 Access-Control-Allow-Origin: *
 ```
 
+## Recommendations
+
+SEP-6 lays out many options for how deposit and withdrawal can work. These are recommendations for getting a wallet or anchor implementation working with minimal effort while providing a great user experience.
+
+### Basic Wallet Implementation
+
+* Identify assets you want to support manually in advance
+* For each asset, use information from its `stellar.toml` and its `/info` endpoint to display useful information to the user like transaction fee structure, types of deposits and withdrawal supported, and details about the anchor like street address.
```

Comment:
Following the previous point I would change "For each asset" to "For each asset the account has a trustline to"

Reply:
They might want to deposit something with no trustline (yet), though.